webgpu.h

```diff
@@ -723,29 +818,29 @@ typedef struct WGPUCompilationMessage {
     uint64_t utf16LinePos;
     uint64_t utf16Offset;
     uint64_t utf16Length;
-} WGPUCompilationMessage;
+} WGPUCompilationMessage WGPU_STRUCTURE_ATTRIBUTE;
 
 typedef struct WGPUComputePassTimestampWrite {
     WGPUQuerySet querySet;
     uint32_t queryIndex;
     WGPUComputePassTimestampLocation location;
-} WGPUComputePassTimestampWrite;
+} WGPUComputePassTimestampWrite WGPU_STRUCTURE_ATTRIBUTE;
 
 typedef struct WGPUConstantEntry {
     WGPUChainedStruct const * nextInChain;
     char const * key;
```

Comment:
we might also want a NONNULL for all of the pointers that aren't nullable?

Reply:
That's would be a lot of noise, but maybe? What do others think of adding a `WGPU_NONNULL`

Reply:
@litherum ?